### ISSUE TITLE
Show error details in CLI when swap fails

### ISSUE THREAD

**Issue by michael1011** [ExchangeUnion/xud#1470]
When a swap fails the CLI output looks like this: `failed to swap 0.0001 LTC with peer order 1a8d3260-7b2c-11ea-908d-0327d32bdd14`

One has to check the the logs to find the reason for which the swap failed:
```
10/04/2020 13:07:50.474 [SWAPS] debug: deal 54141e1fc7df1aa45c7f8542303ed653dac823743a9c84905cb677e9fb530ca8 failed due to UnexpectedClientError: 2 UNKNOWN: insufficient local balance
10/04/2020 13:07:50.475 [ORDERBOOK] error: swap between orders 1a8d3260-7b2c-11ea-908d-0327d32bdd14 & 441d4d90-7b2c-11ea-b47c-67c31ad670ec failed due to UnexpectedClientError
10/04/2020 13:07:50.475 [ORDERBOOK] warn: swap for 10000 failed during order matching due to UnexpectedClientError, will repeat matching routine for failed quantity
```

The CLI should also show the reason for which the swap failed

**Comment by kilrau** [comment]
The print in the example above could read like `failed to swap 0.0001 LTC with peer order 1a8d3260-7b2c-11ea-908d-0327d32bdd14 due to insufficient local balance.`

**Comment by sangaman** [comment]
I believe this was resolved by #1537, which makes it so the output now prints `failed to swap due to [reason]...`
